fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/DerbyBootstrapper.java

```diff
@@ -137,7 +174,7 @@ public static void bootstrap(Connection connection, String adminSchemaName, Stri
             populateResourceTypeAndParameterNameTableEntries(connection, adminSchemaName, dataSchemaName);
         }
     }
-
+    
```

Comment:
Where do these stray tabs come from

Reply:
Eclipse adds them if you look at it in the wrong way

Reply:
removed